### PR TITLE
Document person show endpoint

### PR DESCRIPTION
**Pull request by danidoni**
The OpenAPI documentation for the `person#show` action.

_Replace this line with a description of your changes. Please follow the suggestions in the comment below._

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:

* [ ] Requires a database migration that can cause downtime. Postpone deployment until maintenance window[1].
* [ ] Contains a data migration that can cause temporary inconsistency, so should be run at a specific point of time.
* [ ] Changes some configuration files (e.g. options.yml), so the changes have to be applied manually in the reference server.
* [ ] A new Feature Toggle[2] should be enabled in the reference server.
* [ ] Proper documentation or announcement has to be published upfront since the introduced changes can confuse the users.

[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature

### CODE REVIEW

File: src/api/public/apidocs-new/paths/person.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+get:
+  summary: List all persons.
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/paths/person.yaml:2]: The plural of _person_ is **people**, not _persons_.

**Review by dmarcoux — COMMENTED** [review]
There is a typo in the commit message: _endpoin**t**_ instead of _endpoing_

File: src/api/public/apidocs-new/paths/person.yaml (hunk 1 above)
Comment (danidoni) [reply src/api/public/apidocs-new/paths/person.yaml]: So true :facepalm: Fixed

File: src/api/public/apidocs-new/paths/person.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+get:
+  summary: List all people.
+  description: >
+    List all persons.
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/paths/person.yaml:4]: Power to the people! :wink:

Comment (danidoni) [reply src/api/public/apidocs-new/paths/person.yaml]: True also, fixed

**Comment by danidoni** [comment]
[Person](https://trello.com/c/LrjRFGxI/1596-person)

**Review by eduardoj — APPROVED** [review]
LGTM. I would merge the commits into the first one.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**